### DISCUSSION TITLE
Q-Select loosing focus

### DISCUSSION THREAD

**Discussion by alex9849** [quasarframework/quasar#13334] [General - Components / Directives / etc]
Hi,

I'm currently developing a chrome browser extension, that should add a virtual keyboard. For now almost everything works. I can use the extension to edit all kinds of input fields. Event the ones from quasar.
The only thing that is making problems is the q-select component. I use the q-select component with the lazy loading. That means that I can type something in the select field and the field fetches the possible options from the backend. These options can then be selected from a dropdown menu that appears under the input field.
My problem is that as soon as the menu appears, all my inputs from my virtual keyboard seem to trigger an unfocus of the component. This only happens for this component. For everything else it works.
I use the preventDefault method on the mouseDown-event if I link into the keyboard.

A working component:
https://puu.sh/IYrFg/fa1732c45c.gif

The component that is making problems:
https://puu.sh/IYrEE/a703e135c9.gif

This is my chrome extension script:
https://github.com/alex9849/chrome-simple-keyboard/blob/e9613919fcaab231a5e0bb4c97384ef684c52a3d/src/contentScript.js

Any ideas what could be the problem and how I could prevent that?

**Comment by pdanpdan** [comment]
If you can hook the events in the main document:

- try to listen on main document `focusin` event (in passive capture mode)
- if the event is from somewhere in main document to somewhere outside (your extension document) set a `qClickOutside` key on the event with value `true`

It might work
